arch/arm64/boot/dts/qcom/msm8996.dtsi

```diff
@@ -748,13 +748,19 @@
 				< 1593600 1036000 >;
 		};
 
-		mincpubw-cpufreq {
-			target-dev = <&mincpubw>;
-			cpu-to-dev-map-0 =
-				< 1420800 1525 >;
-			cpu-to-dev-map-2 =
-				< 1420800 1525 >,
-				< 1593600 5195 >;
+		blsp2_spi5: spi@075ba000{
```

Comment:
Leave this as is, we don't have a `<gcc` clock.

Reply:
Think I've done this one as you stated. If not I'll fix it later this afternoon.